### PR TITLE
Configure case type IDs for services (duplicate prevention)

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Configure case type IDs for services, so that duplicate case prevention can work.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -75,16 +75,24 @@ service-config:
...
+        - StandingSearch
+        - WillLodgement
       allow-creating-case-before-payments-are-processed: true
 
     - service: divorce
       jurisdiction: DIVORCE
-      case-type-ids: []
+      case-type-ids:
+        - DIVORCE
+        - DIVORCE_NOTICE_OF_ACTING
```
Comment (lgonczar) [inline src/main/resources/application.yaml]: This one is to be confirmed by Divorce (they're going to come back to us), but for now it's better to stay on the safe side and include it.

**Comment by lgonczar** [comment]
> could you please add change history ?

What do you mean?

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -75,16 +75,24 @@ service-config:
     - service: probate
       jurisdiction: PROBATE
       transformation-url: ${TRANSFORMATION_URL_PROBATE}
-      case-type-ids: []
+      case-type-ids:
+        - GrantOfRepresentation
+        - Caveat
+        - StandingSearch
+        - WillLodgement
```
Comment (rkondratowicz) [inline src/main/resources/application.yaml]: what are `case-type-ids` and what does an empty list mean

Comment (lgonczar) [reply src/main/resources/application.yaml]: There is a comment on the whole configuration section. I've updated it. Not introducing anything in this PR, just concrete values.

**Pull request merged**